src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -933,34 +920,29 @@ await editorSession
                 editorSession.LanguageService.FindSymbolsInFile(
                     scriptFile);
 
-            SymbolInformation[] symbols = null;
-
             string containerName = Path.GetFileNameWithoutExtension(scriptFile.FilePath);
 
+            SymbolInformation[] symbols = s_emptySymbolResult;
             if (foundSymbols != null)
             {
-                symbols =
-                    foundSymbols
-                        .FoundOccurrences
-                        .Select(r =>
-                            {
-                                return new SymbolInformation
-                                {
-                                    ContainerName = containerName,
-                                    Kind = GetSymbolKind(r.SymbolType),
-                                    Location = new Location
-                                    {
-                                        Uri = GetFileUri(r.FilePath),
-                                        Range = GetRangeFromScriptRegion(r.ScriptRegion)
-                                    },
-                                    Name = GetDecoratedSymbolName(r)
-                                };
-                            })
-                        .ToArray();
-            }
-            else
-            {
-                symbols = new SymbolInformation[0];
+                var symbolAcc = new List<SymbolInformation>();
```

Comment:
This all looks great.  Just curious what the `Acc` in `symbolAcc` is short for?  Accumulator?

Reply:
Sorry, accumulator, yes. Habit from writing tail-recursive folds in other languages. Maybe I should change it to something more obvious? Always conscious that having so many variables named essentially the same thing is a code smell, so open to suggestions :)

Reply:
Not a big deal. Just wanted to make sure I understood what it meant.